### PR TITLE
harmonize base controller settings

### PR DESCRIPTION
**Pull request by floweisshardt**
this setup was tested on cob4-2 during store tests in ingolstadt
- unifies joint sequence
- applies controller parameters to all robots

### CODE REVIEW

File: cob_hardware_config/cob4-3/config/base_controller.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## joint_names
-joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint]
+joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint]
```
Comment (fmessmer) [inline cob_hardware_config/cob4-3/config/base_controller.yaml:2]: how did `cob4-3`, `cob4-4`, `cob4-6` work before with the `joint_names` set wrong? (see `use_old_joint_names` in URDF!)

Isn't this in conflict with #447?

File: cob_hardware_config/cob4-2/config/base_driver.yaml [hunk 2]
```diff
@@ -28,7 +28,10 @@ drive_wheel: &drive_wheel
   pos_to_device: "rint(pos * 4096 * 21 * 1 / (2 * pi))" # rev * gear * belt / (2 * pi)
   vel_to_device: "rint(vel * 4096 * 21 * 1 / (2 * pi))" # rev * gear * belt / (2 * pi)
   pos_from_device: "obj6064 * 2 * pi / ( 4096 * 21 * 1)" # 2 * pi / (rev * gear * belt)
-  vel_from_device: "obj606c * 2 * pi / ( 4096 * 21 * 1)" # 2 * pi / (rev * gear * belt)
+  vel_from_device: "p1 != p1 ? 0 : ((obj6064-p1) * 2 * pi / ( 4096 * 21 * 1)) / (0.02*2), p1=p0, p0 = obj6064"
```
Comment (fmessmer) [inline cob_hardware_config/cob4-2/config/base_driver.yaml:31]: why do you need to get velocity derived from position?
after we set the base to `UM=2` (see #445) everything was much smoother...@ipa-mdl

File: cob_hardware_config/cob4-2/config/base_controller.yaml [hunk 3]
```diff
@@ -19,8 +19,8 @@ twist_controller:
       spring: 15.0 #10.0
       damp: 2.75 #2.5
       virt_mass: 0.1 #0.1
-      d_phi_max: 12.0 #100.0
-      dd_phi_max: 17.0 #100.0
+      d_phi_max: 10.0 #100.0
+      dd_phi_max: 40.0 #100.0
```
Comment (fmessmer) [inline cob_hardware_config/cob4-2/config/base_controller.yaml:23]: what do these parameters do?
how were they determined?

**Comment by fmessmer** [comment]
I think this PR is obsolete as it includes "workarounds" that we tried before the proper solution was found, i.e. setting `UM=2` in the firmware (see also https://github.com/ipa320/cob_control/issues/98)

File: cob_hardware_config/cob4-3/config/base_controller.yaml (hunk 1 above)
Comment (floweisshardt) [inline cob_hardware_config/cob4-3/config/base_controller.yaml:2]: #447 doesn't touch this, so no conflict

File: cob_hardware_config/cob4-2/config/base_driver.yaml (hunk 2 above)
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/base_driver.yaml:31]: no idea. This PR contains the status we've tested successfully in ingolstadt

File: cob_hardware_config/cob4-2/config/base_controller.yaml (hunk 3 above)
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/base_controller.yaml:23]: @ipa-mdl magic fingers

File: cob_hardware_config/cob4-3/config/base_controller.yaml (hunk 1 above)
Comment (fmessmer) [inline cob_hardware_config/cob4-3/config/base_controller.yaml:2]: I don't mean merge conflict....I mean the fact, that in #447 @ipa-nhg pushed the version that was used when shipping the platforms

File: cob_hardware_config/cob4-2/config/base_controller.yaml (hunk 3 above)
Comment (mathias-luedtke) [inline cob_hardware_config/cob4-2/config/base_controller.yaml:23]: These are the velocity (`d_phi_max`) and acceleration (`dd_phi_max`) limits of the steer controller.
(https://github.com/ipa320/cob_robots/blob/hydro_dev/cob_hardware_config/cob4-2/config/base/MotionCtrl.ini#L64)

File: cob_hardware_config/cob4-2/config/base_driver.yaml (hunk 2 above)
Comment (mathias-luedtke) [inline cob_hardware_config/cob4-2/config/base_driver.yaml:31]: This was a test, not sure if needed.. We should stick to the new filter feature if we need averaging.

File: cob_hardware_config/cob4-3/config/base_controller.yaml (hunk 1 above)
Comment (mathias-luedtke) [inline cob_hardware_config/cob4-3/config/base_controller.yaml:2]: The `joint_names` parameter is not used by the driver or the controllers.. I believe it is used by `cob_script_server`/`cob_command_gui`

**Comment by fmessmer** [comment]
@ipa-fmw @ipa-nhg is this used on the robot account?
Could you test the robot account without this PR (i.e. using ipa320/cob_robots indigo_dev) and see whether performance is fine?

**Comment by fmessmer** [comment]
when testing it together with @fmw-hb yesterday, behavior without this PR is fine 
(it even passed @ipa-tif's escalation test :wink:)

**Pull request closed without merging**